### PR TITLE
Revised Definition for Diversity (Issue 342)

### PR DESCRIPTION
**Pull request by wareid**
Based on discussion in issue #342 and in the meeting on 29 Aug, here is the proposed text for the new definition for diversity in the glossary.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/pull/345.html" title="Last updated on Oct 6, 2023, 8:08 PM UTC (642dbae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/345/78e8d8a...642dbae.html" title="Last updated on Oct 6, 2023, 8:08 PM UTC (642dbae)">Diff</a>

### CODE REVIEW

File: index.html
Comment (nigelmegitt) [inline index.html:613]: Having read the record of the meeting at https://www.w3.org/2023/08/29-pwe-minutes.html I would +1 changing "race" to "ascribed race".

Comment (dbooth-boston) [reply index.html:613]: I don't think it's appropriate for the link for "indigeneity" to be a google search.  Please change it to link to a dictionary definition.

**Comment by wareid** [comment]
It is a reference to oxford dictionaries, I do not have a subscription to be able to access the definition otherwise. We use this style of link elsewhere too.

File: index.html
Comment (cwilso) [reply index.html:613]: I might suggest https://www.merriam-webster.com/dictionary/indigenous.  If you do want to stick with the Google search, remove the extraneous stuff:  https://www.google.com/search?q=indigeneity.

Comment (TzviyaSiegman) [inline index.html:613]: We agreed in today's meeting to change the URL to the one Chris suggested https://www.google.com/search?q=indigeneity

**Review by TzviyaSiegman — CHANGES_REQUESTED** [review]

File: index.html
Comment (dbooth-boston) [reply index.html:613]: Now that the extraneous stuff has been removed from the google search URL I can see more clearly what the URL is doing.   And it clearly is NOT a link to a definition.  It is a plain vanilla search that happens to currently show a definition of the term.  That could change any moment if google decided to change the design of its search results page.

I think it would be much better to either link to an actual dictionary definition (that is accessible without a paywall) or include our own definition in the glossary.  We owe it to our readers to define our jargon or point to a stable, accessible definition.

Comment (dbooth-boston) [reply index.html:613]: I tried to find a better link to suggest, but did not find "indigeneity" in any dictionary other than Oxford Online, which has a paywall.  But I did notice that "indigenousness" appears in all major dictionaries, and as far as I can tell, it means the same thing: the quality of being indigenous.   Indeed, if "indigeneity" was intended to mean something subtly different than "the quality of being indigenous" then I think that would be a problem because our documents should be for a broad audience.   I also see that there is [significant debate](https://journals.sagepub.com/doi/abs/10.1177/1177180119828380) about the meaning of "indigeneity" in the academic literature, presumably because of attempts to define the term in specialized ways in legal contexts.

Based on this research, I suggest changing "indigeneity" to "indigenousness".  As far as I can tell, that would still convey the meaning that we are trying to convey, without using a contested term that is not well established in English.

Comment (dbooth-boston) [reply index.html:613]: I don't want to hold up publication, so I suggest we go ahead and merge using the google link as decided on today's call.  But I think it would be good to improve this in the future -- next year? -- and I think the research I turned up qualifies as sufficient new information to re-open the topic for consideration in a future revision of the document.  I can open it as a new issue after you have merged, and note in the issue that it is not intended for immediate consideration.

**Pull request merged**
